wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/FieldExtensions.kt

```diff
@@ -23,10 +23,10 @@ import com.squareup.wire.schema.Field
 import com.squareup.wire.schema.ProtoType
 
 internal val Field.isMap: Boolean
-  get() = type().isMap
+  get() = type!!.isMap
```

Comment:
... might change this to be lateinit later. It never escapes the Linker without being non-null.